pynumdiff/utils/evaluate.py

```diff
@@ -76,8 +76,9 @@ def plot_comparison(dt, dxdt_truth, dxdt_hat1, title1, dxdt_hat2, title2, dxdt_h
     fig.tight_layout()
 
 
-def metrics(x, dt, x_hat, dxdt_hat, x_truth=None, dxdt_truth=None, padding=0):
-    """Evaluate x_hat based on various metrics, depending on whether dxdt_truth and x_truth are known or not.
+def rmse(x, dt, x_hat, dxdt_hat, x_truth=None, dxdt_truth=None, padding=0):
```

Comment:
I decided to rename this one, because all the metrics it's calculating are RMSE-related.